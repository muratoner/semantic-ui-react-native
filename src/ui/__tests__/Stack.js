import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import React from 'react'
import {StyleSheet} from 'react-native'
import {Stack} from '../../'

describe('Stack Component', () => {
	it('should render without issues', () => {
		const component = shallow(
			<Stack size={10} style={{backgroundColor: 'red'}} />
		)

		expect(component.length).toBe(1)
		const style = StyleSheet.flatten(component.get(0).props.style)
		expect(style).toHaveProperty('height', 10)
		expect(style).toHaveProperty('backgroundColor', 'red')
		expect(toJson(component)).toMatchSnapshot()
	})
})
