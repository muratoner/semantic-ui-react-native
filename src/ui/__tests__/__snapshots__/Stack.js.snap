// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stack Component should render without issues 1`] = `
<View
  style={
    Array [
      Object {
        "height": 10,
      },
      Object {
        "backgroundColor": "red",
      },
    ]
  }
/>
`;
